packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -504,6 +504,7 @@ library. For the currently supported features, we will include the following lib
 -   [@tiptap/extension-history](https://www.npmjs.com/package/@tiptap/extension-history)
 -   [@tiptap/extension-italic](https://www.npmjs.com/package/@tiptap/extension-italic)
 -   [@tiptap/extension-list-item](https://www.npmjs.com/package/@tiptap/extension-list-item)
+-   [@tiptap/extension-list-keymap](https://tiptap.dev/api/extensions/list-keymap)
```

Comment:
nit: Can we link the npm site to this package, similar to how other packages are linked? This provides information about the dependency packages like the size, license, etc., 

```suggestion
-   [@tiptap/extension-list-keymap](https://www.npmjs.com/package/@tiptap/extension-list-keymap)
```